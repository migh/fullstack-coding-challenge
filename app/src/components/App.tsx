import * as React from 'react';

import { InputContainer } from './Input.container';

export const App = () => (
  <div id="container">
    <div className="row justify-content-md-center">
      <div className="col-12 col-lg-6 text-center">
        <h2>Test Unbabel API</h2>
        <InputContainer />
      </div>
    </div>
  </div>
);
