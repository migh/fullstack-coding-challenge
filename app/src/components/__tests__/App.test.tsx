import * as React from 'react';
import { shallow } from 'enzyme';

import { App } from '../App';

describe('<App />', () => {
  describe('Snapshots', () => {
    it('should render.', () => {
      expect(shallow(<App />)).toMatchSnapshot();
    });
  });
});
