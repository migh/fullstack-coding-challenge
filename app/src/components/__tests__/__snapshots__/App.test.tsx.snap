// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Snapshots should render. 1`] = `ShallowWrapper {}`;
